kbase-extension/scss/partials/_fileUploadWidget.scss

```diff
@@ -0,0 +1,59 @@
+.kb-file-upload-widget {
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    margin-bottom: 0.5rem;
+
+    &__error-message {
+        display: flex;
+        flex-direction: row;
+
+        .-title {
+            flex: 0 0 4rem;
+            align-items: center;
+        }
+
+        .-body {
+            flex: 1 1 0;
+            display: flex;
+            flex-direction: column;
+        }
+    }
+
+    &__clear-all-button-container {
+        margin-top: 0.5rem;
+    }
+
+    &__globus-upload-link-container {
+        display: inline-block;
+        vertical-align: baseline;
+    }
+
+    // Bootstrap overrides
+
+    // necessary?
+    .progress-bar {
```

Comment:
It might not be necessary at this point, but Bootstrap's progressbar is occasionally an oddball, and was annoying to work against Dropzone.

Reply:
bootstrap already sets the width for .progress-bar to 0, and I don't see anywhere in the KBase codebase that sets this class otherwise that would need overriding.

Also, in the widget code the width is set via direct style to '0' on reset, but should probably set it to '' to remove the style and let the class style take over.

Anyway, it is working, so I don't see any reason to change anything here, other than the fact that these files are being revised and who knows when the next opportunity will come.